src/librustdoc/html/static/css/themes/ayu.css

```diff
@@ -9,6 +9,11 @@ Original by Dempfi (https://github.com/dempfi/ayu)
 	--settings-input-color: #ffb454;
 	--sidebar-background-color: #14191f;
 	--sidebar-background-color-hover: rgba(70, 70, 70, 0.33);
+	--code-block-background-color: #191f26;
+	--scrollbar-track-background-color: transparent;
+	--scrollbar-thumb-background-color: #5c6773;
+	--scrollbar-color: #5c6773 #24292f;
```

Comment:
This shows an interesting inconsistency. I would expect `transparent` to be `#24292f` or vice verse. Any idea why there's a difference?

Reply:
> I would expect `transparent` to be `#24292f` or vice verse.

Sorry I'm a bit lost on this part. Could you explain a bit more please? 😅

Reply:
Oh I just understood. I'm not too sure to be honest. It seems a bit weird indeed.